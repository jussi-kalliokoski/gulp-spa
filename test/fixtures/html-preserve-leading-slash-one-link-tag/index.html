<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <!-- build:css -->
    <link rel="stylesheet" href="/file1.css" />
    <link rel="stylesheet" href="/file2.css" />
    <!-- endbuild -->
</head>
<body>
</body>
</html>
