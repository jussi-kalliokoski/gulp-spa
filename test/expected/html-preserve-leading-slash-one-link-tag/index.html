<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <link rel="stylesheet" href="/joined.css" />
</head>
<body>
</body>
</html>
